external/model-preparation-algorithm/configs/detection/mobilenetv2_ssd_cls_incr/model.py

```diff
@@ -0,0 +1,8 @@
+_base_ = [
+  '../../../submodule/samples/cfgs/models/backbones/ote_mobilenet_v2_w1.yaml',
+  '../../../submodule/recipes/stages/_base_/models/detectors/ssd.custom.py'
+]
+
+model = dict(
+    backbone=dict(out_indices=(4, 5,))
```

Comment:
What does this mean?

Reply:
I just follow ATSS's model.py, mmdetection/configs/custom-object-detection/gen3_mobilenetV2_SSD/model.py.
I guess this code may specify which backbone's features would be used by neck or bbox_head.